js/index.js

```diff
@@ -1 +1,100 @@
 // Your code here
+function diceRandom(num) {
+    var theRoll = Math.floor(Math.random() * (num) + 1);
+    if (theRoll > 0) {return theRoll;}
+
+}
+
+var diceRoll = diceRandom(6);
+
+console.log(diceRoll);
+
+var diceSquareOne = document.getElementById('one');
+var diceSquareTwo = document.getElementById('two');
+var diceSquareThree = document.getElementById('three');
+var diceSquareFour = document.getElementById('four');
+var diceSquareFive = document.getElementById('five');
+var diceSquareSix = document.getElementById('six');
+var diceSquareSeven = document.getElementById('seven');
+var diceSquareEight = document.getElementById('eight');
+var diceSquareNine = document.getElementById('nine');
+
```

Comment:
Good thinking, but these functions are doing a little too much and this could definitely be done with less code.